src/test/java/com/duckblade/osrs/dpscalc/calc/gearbonus/VampyreGearBonusTest.java

```diff
@@ -16,23 +27,67 @@
 import static org.mockito.Mockito.when;
 import org.mockito.junit.jupiter.MockitoExtension;
 
+import java.util.Collections;
+
 @ExtendWith(MockitoExtension.class)
 class VampyreGearBonusTest
 {
+	@Mock
+	private WeaponComputable weaponComputable;
 
 	@Mock
 	private ComputeContext context;
 
 	@InjectMocks
 	private VampyreGearBonus vampyreGearBonus;
 
+	@Test
+	void isApplicableWhenWearingIvandis()
```

Comment:
`isApplicableWhenUsingVampyrebaneAgainstVampyres` and merge the weapon list into one test case similar to https://github.com/LlemonDuck/dps-calculator/blob/f329112ea9bdb243e206f93a26a4d4e818742402/src/test/java/com/duckblade/osrs/dpscalc/calc/gearbonus/DragonHunterGearBonusTest.java#L57-L69